blog/_posts/2017-11-21-Team- Alexa-at-Web-Summit.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Team Alexa at Web Summit
+layout: post
+created_at: Tue November 21 2017
```

Comment:
Hi, thanks for the PR. Can you just change the date everywhere to today [Mon November 27 2017]?